freelancer-rates/lib/freelancer_rates.ex

```diff
@@ -0,0 +1,22 @@
+defmodule FreelancerRates do
+  def daily_rate(hourly_rate) do
+   hourly_rate * 8.0
+  end
+
+
+  def apply_discount(before_discount, discount) do
+      before_discount - (before_discount * discount/100)
```

Comment:
```suggestion
      before_discount - (before_discount * discount / 100)
```
I generally don't like to perform arithmetic operations without a space in between the operator.